test/functional/legacy/increment_spec.lua

```diff
@@ -1,723 +1,742 @@
 -- Tests for using Ctrl-A/Ctrl-X on visual selections
 
 local helpers = require('test.functional.helpers')
+local assert = require('luassert')
 local source, execute = helpers.source, helpers.execute
+local feed, eval, funcs = helpers.feed, helpers.eval, helpers.funcs
 local call, clear = helpers.call, helpers.clear
 local eq, nvim = helpers.eq, helpers.meths
 
+local getline, cursor, getpos = funcs.getline, funcs.cursor, funcs.getpos
+local setline = funcs.setline
+
+local function repeater(string, number)
+  assert(0 < number)
+  local t = {}
+  local i = 0
+
+  for i = 1, number do
+    t[i] = string
+  end
+
+  return t
+end
+
 describe('Ctrl-A/Ctrl-X on visual selections', function()
+  before_each(clear)
+
+  -- 1) Ctrl-A on visually selected number
+  -- Text:
+  -- foobar-10
+  --     Expected:
+  --     1)    Ctrl-A on start of line:
+  --     foobar-9
+  --     2)    Ctrl-A on visually selected "-10":
+  --     foobar-9
+  --     3)    Ctrl-A on visually selected "10":
+  --     foobar-11
+  --     4)    Ctrl-X on visually selected "-10"
+  --     foobar-11
+  --     5)    Ctrl-X on visually selected "10"
+  --     foobar-9
+  it('Test_visual_increment_01', function()
+    setline(1, repeater("foobaar-10",5))
```

Comment:
If I'm understanding helpers.call correclty, couldn't you replace `repeater("foobar-10",5)` with `call('repeat', {'foobar-10'}, 5)`?